### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by marwinpg**
This branch has the changes related to the content of the portfolio.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,27 +1,35 @@
 <!DOCTYPE html>
+<!-- <html lang="es"> -->
 <html>
     <head>
+        <!-- <meta charset="utf-8" /> -->
```
Comment (marwin-sfdc) [inline index.html:5]: Don't forget to remove the comment.

File: index.html [hunk 2]
```diff
@@ -1,27 +1,35 @@
 <!DOCTYPE html>
+<!-- <html lang="es"> -->
```
Comment (marwin-sfdc) [inline index.html:2]: Don't forget to remove the comment

File: index.html (hunk 1 above)
Comment (marwinpg) [reply index.html]: Fixed!

File: index.html (hunk 2 above)
Comment (marwinpg) [reply index.html]: Fixed!

**Pull request merged**